frontend/src/components/SymtomEntryOverview.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { useNavigate } from "react-router";
+import { useSearchParams } from "react-router-dom";
+import Sidebar from "./Sidebar";
+import {HeaderText} from './nextUI/index';
+import {
+  fetchAllSymptomEntries,
+  selectAllSymptoms,
+  deleteSymptomEntry,
+} from "../store/allEntriesSlice";
+import { v4 as uuidv4 } from "uuid";
+
+function SymptomEntryOverview() {
+  const [symptoms, setSymptoms] = useState([]);
+  const [count, setCount] = useState(0);
+  const symptomEntries = useSelector(selectAllSymptoms);
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const maxPages = Math.ceil(count / 20);
+
+  const [searchParams] = useSearchParams();
+  let { page } = Object.fromEntries([...searchParams]);
+  page = Number(page);
+  if (!page || isNaN(page)) page = 1;
+
+  useEffect(() => {
+    dispatch(fetchAllSymptomEntries(page));
+    navigate(`/user/symptom-entries?page=${page}`);
+  }, [dispatch, page]);
+
+  useEffect(() => {
+    setCount(symptomEntries.symptomCount);
+    setSymptoms(symptomEntries.symptoms);
+  }, [symptomEntries]);
+
+  const handlePageChange = (event) => {
+    if (event.target.value === "previous") {
```

Comment:
It looks like you pass in a second argument when you call this fn, in which case you can just use that parameter instead of `event.target.value`